app/javascript/components/batch_update/components/SearchForm.js

```diff
@@ -121,7 +132,12 @@ class SearchForm extends React.Component {
             createdBeforeDate={createdBeforeDate}
           />
         )}
-
+        <PriceInput
+          minPrice={minPrice}
+          maxPrice={maxPrice}
+          onAddMinPrice={onAddMinPrice}
+          onAddMaxPrice={onAddMaxPrice}
+        />
```

Comment:
I suppose the comment about hiding these inputs could apply here instead — we could follow the pattern that's being used for the date inputs above.